collections.js

```diff
@@ -118,8 +118,79 @@ LinkedList.prototype.find = function(value) {
     node = node.next;
   }
   return null;
+};
+
+//------------------------------------------------------------------------------
+// Simple Queue - a trivial queue implementation used to verify tests.
+// The head of the queue is at the beginning of the backing array.
+
+function SimpleQueue() {
```

Comment:
I was thinking the same thing.  It should really be part of the tests, if
indeed it's still useful for testing.  It was for a while, but probably no
longer useful.

We'll talk more tomorrow!

On Mon, Feb 29, 2016 at 11:06 AM, billbudge <notifications@github.com>
wrote:

> In collections.js
> <https://github.com/billbudge/Diagrammar/pull/2#discussion_r54459708>:
>
> > @@ -118,8 +118,79 @@ LinkedList.prototype.find = function(value) {
> >      node = node.next;
> >    }
> >    return null;
> > +};
> > +
> > +//------------------------------------------------------------------------------
> > +// Simple Queue - a trivial queue implementation used to verify tests.
> > +// The head of the queue is at the beginning of the backing array.
> > +
> > +function SimpleQueue() {
>
> It's weird to have this exposed as part of the API, since it's not useful
> as it stands. You don't appear to use it in your implementation of Queue
> either.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/billbudge/Diagrammar/pull/2/files#r54459708>.
>